trade-finance-manager-ui/server/controllers/utilisation-reports/utilisation-report-reconciliation-for-report/extract-query-and-session-data.test.ts

```diff
@@ -7,59 +7,61 @@ jest.mock('./validate-facility-id-query');
 
 describe('extractQueryAndSessionData', () => {
   const ORIGINAL_URL = '/original-url';
-  const FACILITY_ID_QUERY = '1234';
+  const PREMIUM_PAYMENTS_FACILITY_ID_QUERY = '1234';
 
   beforeEach(() => {
     jest.resetAllMocks();
 
     jest.mocked(handleRedirectSessionData).mockReturnValue({
-      tableDataError: undefined,
+      premiumPaymentsTableDataError: undefined,
       selectedFeeRecordIds: new Set(),
     });
   });
 
-  it('uses provided facilityIdQuery', () => {
-    // Arrange
-    const queryParams = {
-      facilityIdQuery: FACILITY_ID_QUERY,
-    };
-    const sessionData = {};
+  describe('premium payments filters', () => {
+    it('uses provided premiumPaymentsFacilityId', () => {
+      // Arrange
+      const queryParams = {
+        premiumPaymentsFacilityId: PREMIUM_PAYMENTS_FACILITY_ID_QUERY,
```

Comment:
What's the reason for renaming the query param please? I think `facilityId` is simpler and cleaner than `premiumPaymentsFacilityId`.

I'm don't know the URL off the top of my head, but i'm imagining with the current changes it will be something like this:

`dtfs.com/utlisation-report-reconciliation?premiumPaymentsFacilityId=1234,5678`

Where IMO this would be cleaner:

`dtfs.com/utlisation-report-reconciliation?facilityIds=1234,5678`

Reply:
We have different tabs on this page, each with their own filters (some of which filter the same attributes). Previously we only had filters on the premium payments tab, but now we've also added them to to the payment details tab. Both sets of these filters are passed through as query parameters so the prefix has been added to distinguish which tab the filter applies to - hope that makes sense

![Screenshot 2024-10-01 at 11 32 39](https://github.com/user-attachments/assets/f2efb2d8-c842-4795-b366-089cbb1bf5fb)

![Screenshot 2024-10-01 at 11 32 32](https://github.com/user-attachments/assets/98178b82-d620-4505-b0da-76d3b7c7de69)

Reply:
Great thanks for the explanation @sw-mattw makes sense 👍